regularizepsf/fitter.py

```diff
@@ -22,8 +22,9 @@
 
 
 class PatchCollectionABC(metaclass=abc.ABCMeta):
-    def __init__(self, patches: dict[Any, np.ndarray]) -> None:
+    def __init__(self, patches: dict[Any, np.ndarray], counts: dict[Any, int] = None) -> None:
```

Comment:
I think the type hint of `counts` should be `Optional` since we're setting it to `None`.